cbmackintosh/intention-timer
# Feature/update readme lastday

*What does this PR do?*
- feature adding sounds to start and complete

**Is this a feature or refactor?**
- feature

**How should this be tested?**
- Should hear a click sound when you click start and clapping when the timer reaches zero
